tests/test_abcrules.py

```diff
@@ -693,6 +694,7 @@ def remove_solver_output(out):
         "Set parameter LicenseID to value .*\n",
         "Academic license - for non-commercial use only.*\n",
         "Restricted license - for non-production use only - expires.*\n",
+        "HighsMipSolverData::transformNewIntegerFeasibleSolution tmpSolver.run();\n",
```

Comment:
since this is a regexp, the . is probably causing problems. Maybe simply replace with "HighsMipSolverData::.*\n"